api/public/schema/userSchema.js

```diff
@@ -0,0 +1,53 @@
+const schema = {
+  id: "/UserSchema",
+  type: "object",
+  properties: {
+    email: {
+      type: "string",
+      required: true
+    },
+    username: {
+      type: "string",
+      required: true
+    },
+    password: {
+      type: "string",
+      required: true
+    },
+    country: {
+      type: "string",
+      required: true
+    },
+    birthday: {
+      type: "string",
+      required: true
+    },
+    role: {
+      type: "string",
+      enum: ["student", "teacher", "stakeholder"],
+      required: true
+    },
+    admin: {
+      type: "boolean"
+    },
+    anon: {
+      type: "boolean"
+    },
+    projectIds: {
```

Comment:
This could be renamed to createdProjects to be more explicit.

Reply:
Good point ✏️